src/components/AppBar/index.scss

```
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '@material/tab/mixins';
+@import '@material/tab-indicator/mixins';
+
+.AppBar {
+  padding: 0 24px;
+
+  .title-grid {
+    align-items: center;
+    align-self: flex-end;
+    display: grid;
+    grid-gap: 4px;
+    grid-template-areas: 'logo title'
+                         '.    subtitle';
+
+    .title {
+      grid-area: title;
+    }
+
+    .subtitle {
+      grid-area: subtitle;
+    }
+
+    svg {
+      grid-area: logo;
+    }
+  }
+
+  .mdc-tab-bar {
+    padding-left: 24px;
+  }
+
+  .mdc-tab {
+    @include mdc-tab-active-text-label-color('secondary');
+    @include mdc-tab-horizontal-padding(5px);
```

```suggestion
    @include mdc-tab-horizontal-padding(.25rem);
```

not exact math but should work fine.

---

see comments below on rem vs px